server/cache.go

```diff
@@ -369,8 +371,16 @@ func (c *clusterInfo) handleStoreHeartbeat(stats *pdpb.StoreStats) error {
 	if store == nil {
 		return errors.Trace(core.ErrStoreNotFound(storeID))
 	}
+	lastStats := store.Stats
 	store.Stats = proto.Clone(stats).(*pdpb.StoreStats)
 	store.LastHeartbeatTS = time.Now()
+	// denosing
+	if store.Stats.GetInterval().GetEndTimestamp()-store.Stats.GetInterval().GetStartTimestamp() < minStoreHeartbeatInterval {
+		store.Stats.BytesRead = lastStats.BytesRead
+		store.Stats.BytesWritten = lastStats.BytesWritten
+		store.Stats.KeysRead = lastStats.KeysRead
+		store.Stats.KeysWritten = lastStats.KeysWritten
```

Comment:
Do we need to transform the store's flow bytes to bytes per second using the interval reported by tikv? Do we need to use the _RollingStatistics_ like region flow bytes?